cmd/main.go

```diff
@@ -0,0 +1,211 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"os/user"
+	"path"
+	"runtime"
+	"strings"
+
+	"github.com/google/go-github/github"
+)
+
+var errorPlatformNotSupported error = errors.New("platform not supported, only supporting windows and drawin")
+
+func IsInstalled() (bool, error) {
+	execFilePath, execFileErr := getExecutableFile()
```

Comment:
NIT: It's idiomatic in go to just call the error err
```suggestion
	execFilePath, err := getExecutableFile()
```